### ISSUE TITLE
update

### ISSUE THREAD

**Issue by labr0110** [labr0110/andreane-labrosse.ca#2]
Hi,

I saw that you could add files to your repository and thought that if I can upload all the missing files it would work, but still nothing.

Andy
@thomasjbradley

**Comment by thomasjbradley** [comment]
Have you cloned this repository to your computer? Or are you working out of a folder you created?

**Comment by labr0110** [comment]
I cloned it!

I had already created a folder called "coming soon" and created a new repository called "andreane-labrosse.ca" which is this one. I copied all the folders that were in my initial folder to the repository. Is that maybe why?

**Comment by thomasjbradley** [comment]
On your computer, if you look in the `andreane-labrosse.ca` folder, all the files are there? Or they're only visible on the GitHub website.

**Comment by labr0110** [comment]
They're all there. Here's a screenshot of what I see.

![screen shot 2017-09-21 at 1 08 20 pm](https://user-images.githubusercontent.com/16653418/30708772-fe4b87c4-9ecd-11e7-9778-4da83f94f79f.png)

**Comment by thomasjbradley** [comment]
Okay, what does it look like in the GitHub application?

**Comment by labr0110** [comment]
![screen shot 2017-09-21 at 1 11 19 pm](https://user-images.githubusercontent.com/16653418/30708903-675f0240-9ece-11e7-9ad2-4eaf46d0704a.png)
![screen shot 2017-09-21 at 1 11 25 pm](https://user-images.githubusercontent.com/16653418/30708904-67702f52-9ece-11e7-881b-d358a1ad0686.png)

**Comment by thomasjbradley** [comment]
Hmm... I think because the files were uploaded in the GitHub interface there's going to be a major conflict. I think you should come see me if you can. Are you at school right now? I'm in my office.

**Comment by labr0110** [comment]
I'm not at school today unfortunately. Will you be there tomorrow? I have class from 8-11.

**Comment by thomasjbradley** [comment]
Yeah, if you come right at 11 that will work. I also have class from 8-11 but I have an appointment to get to.

**Comment by labr0110** [comment]
Okay, I will come right at 11, thank you! Where is your office located again?

**Comment by thomasjbradley** [comment]
J104, under the stairs.

**Comment by labr0110** [comment]
Perfect, see you tomorrow!
